test/tests/link.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const { ariaTest } = require('..');
+
+let pageExamples = [
+  {
+    exampleId: 'ex1',
+    linkSelector: '#ex1 span'
+  },
+  {
+    exampleId: 'ex2',
+    linkSelector: '#ex2 img',
+    alt: true
+  },
+  {
+    exampleId: 'ex3',
+    linkSelector: '#ex3 span',
+    ariaLabel: true
+  }
+];
+
+ariaTest('link/link.html', 'link-role', async (t) => {
+
+  for (let i = 0; i < pageExamples.length; i++) {
+    let ex = pageExamples[i];
+    let linkLocator = t.context.By.css(ex.linkSelector);
+    let linkElement = await t.context.session.findElement(linkLocator);
+
+    t.is(
+      await linkElement.getAttribute('role'),
```

Comment:
Rather than a raw await here, can we provide a utility function that affords an error boundary for unresolved promises? 

I'm thinking ahead to future maintenance. Unresolved promise errors are a devil to debug, so we can catch them at a boundary and emit a human-readable warning, at least we might know where to start looking. Something to ponder, at least. It may not be an ideal approach in this case.